ffi/src/winscard/pcsc_lite/functions.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(all(target_os = "windows", feature = "scard"))]
```

Comment:
this module is supposed to be compiled on Linux and macOS. Why did you make it Windows-only?

Reply:
> this module is supposed to be compiled on Linux and macOS

the idea is to be able to use system-provided smart cards on Linux and macOS using the pcsc-lite library